app/components/object-inspector/accordion.js

```diff
@@ -4,12 +4,25 @@ import Component from '@ember/component';
 export default Component.extend({
   tagName: '',
 
-  isExpanded: computed('mixin.expand', 'mixin.properties.length', function () {
-    return this.get('mixin.expand') && this.get('mixin.properties.length') > 0;
+  _mixin: null,
+
+  isExpanded: undefined,
+
+  mixin: computed({
```

Comment:
I'm curious what this `mixin` computed is for? Aren't we passing `mixin` in here? https://github.com/emberjs/ember-inspector/pull/1035/files#diff-8ce1165ef4c2dc2a8fe46fd33be76df5R2

Reply:
mmm, it might not be needed anymore.
It was an early attempt to fix issues with rerenders from here:
https://github.com/emberjs/ember-inspector/pull/1035/files/96efd0d7e7e98103fcd44e6a696b238d57d6e3df#diff-a9a19ee37ba8619bc68db992b637dd05R87
where the view was reset on every time, with this workaround i was able to keep the current accordion open, but I think it's really fixed by the above linked fix. I test again and revert if this is working

Reply:
This comment appears to be a copied/pasted response for the one about the `key`? This is a different issue.